requirements.txt

```diff
@@ -1,4 +1,8 @@
+# Note: this is assuming a Linux-based installation, for other platforms see https://pytorch.org/get-started/locally/
+
 # --------- pytorch --------- #
+--extra-index-url https://download.pytorch.org/whl/cu118  # gpu+cpu installation (comment this line if doing cpu-only)
+# --extra-index-url https://download.pytorch.org/whl/cpu  # cpu-only installation (uncomment line if doing cpu-only)
```

Comment:
Looking at the https://pytorch.org/get-started/locally/, only cuda 11.8 requires those lines, while 11.7 doesn't, and 11.7 is set as default when installing through normal `pip install torch`.

I think there's not much point in trying to maintain those instructions for `requirements.txt` as they seem to regulary change every couple of weeks/months. I would rather just leave this file simple and let people figure out what they need by themselves

Reply:
Yeah you're right. Perhaps I can add a comment that at least notifies the user of this, so it's not too painful for them to modify requirements.txt or environment.yaml